src/test/scala/rxscalajs/ObservableTest.scala

```diff
@@ -7,11 +7,12 @@ import utest._
 
 import scala.concurrent.{ExecutionContext, Future}
 import scala.scalajs.js
+import scala.scalajs.js.|
 
 
 object ObservableTest extends TestSuite {
 
-
+  type Creator = Unit | (() => Unit)
```

Comment:
Actually, you can keep it this way in the Observable.scala file, but not in the ObservableFacade.scala file. :)